error-codes.md

```diff
@@ -133,102 +139,96 @@ In addition, TiDB has the following unique error codes:
 
     The field fails to obtain the default value. This error is usually used internally, and is converted to a specific type of error for external applications.
 
-* Error Number: 8039
-
-    Index offset is out of range.
-
-* Error Number: 8042
-
-    The state of table schema does not exist.
-
-* Error Number: 8043
-
-    The state of column information does not exist.
-
-* Error Number: 8044
-
-    The state of index does not exist.
-
-* Error Number: 8045
-
-    Invalid table data.
+* Error Number: 8040
 
-* Error Number: 8046
-
-    The state of column information is invisible.
+    Unsupported operations are performed. For example, operating lock table on View and Sequence.
```

Comment:
```suggestion
    Unsupported operations are performed. For example, you perform a table locking operation on a view or a sequence.
```

Reply:
Shall we use uppercase? ( View/Sequence）

Reply:
If it's mentioned in the main body as a noun, the lowercase will be fine. See https://docs.pingcap.com/tidb/stable/sql-statement-create-view#create-view.